lib/alexandria/ui/listview.rb

```diff
@@ -204,7 +204,7 @@ def setup_listview_columns_visibility
         ]
         cols = @listview.columns[1..-1] # skip "Title"
         cols.each_index do |i|
-          cols[i].visible = cols_visibility[i]
+          cols[i].visible = !cols_visibility[i].nil?
```

Comment:
This will make the visibility true even if `cols_visibility[i]` is false.
How about this instead?

```suggestion
          cols[i].visible = !!cols_visibility[i]
```

Reply:
Yes, you are right. I admit that I did it in about two minutes late at night. :-)

I think that there must be a better way to do this anyway.

Reply:
For now this is fine I think. Thanks for looking into this!

Reply:
I have changed it to your suggestion.

There are probably lots of other problems like this. It appears that settings in `Gtk` cannot be `nil` in the latest version of the `glib2` gem (although it was permitted in previous versions). At first i thought it was only boolean values but I have found another one at line 69 of lib/alexandria/ui/book_properties_dialog.rb, which requires a string.